gui/util/misc.py

```diff
@@ -99,8 +99,10 @@ def compute_max_shape(shapes, max_dims=None):
                 if dim_len > max_shape[dim]:
                     max_shape[dim] = dim_len
 
-    return tuple(max_shape)
-
+    if max_shape:
+        return tuple(max_shape)
```

Comment:
This will always be returned, because a non-empty list is truthy, regardless of the values it contains.

Reply:
Ok, I have removed this - it relates to the corner case I mentioned above where we launch a window with an empty viewer and then want to add a new empty markers layer for annotation.